ui/src/scrapers/components/CreateScraperForm.tsx

```diff
@@ -1,14 +1,14 @@
 // Libraries
-import React, {PureComponent, ChangeEvent} from 'react'
+import React, {PureComponent, ChangeEvent, FormEvent} from 'react'
```

Comment:
I'm not sure the type of the `onSubmit` function in `<Form />` but this is making me think it should make use of this `FormEvent` type